database/helper.js

```diff
@@ -22,9 +24,7 @@ module.exports.addSearchQuery = (obj) => {
   const {
     searchQueryId, timestamp, market, checkIn, checkOut, roomType,
   } = obj;
-  const queryText = `INSERT INTO listings (search_id, market, searched_at, check_in, check_out, room_type) 
-    VALUES (${searchQueryId}, '${market}', '${timestamp}', '${checkIn}', '${checkOut}',  '${roomType}') 
-    ON CONFLICT (search_id) DO NOTHING`;
+  const queryText = `INSERT INTO listings (search_id, market, searched_at, check_in, check_out, room_type) VALUES (${searchQueryId}, '${market}', '${timestamp}', '${checkIn}', '${checkOut}',  '${roomType}') ON CONFLICT (search_id) DO NOTHING`;
```

Comment:
Are you planning on getting _search_id_ and _searched_at_ info from Search service or User Behavior service?

Reply:
These ones are from Search Service. From User Behavior I should only get the Search_Id (or Inventory should get it, which is then augmented with Inventory data).